From fba7d9638ebd1cdba8e20cd1a2e4e6c4927435aa Mon Sep 17 00:00:00 2001
From: Pankaj Dev <pankaj.dev@st.com>
Date: Mon, 18 Jul 2022 16:19:40 +0530
Subject: [PATCH 0129/1141] arm64: defconfig: Enable USB2Host and USB3DRD for
 stm32mp25

Enable USB2 host and DWC3 glue drivers for stm32mp25, and related
USB2 PHY used for both controllers.
Note: ComboPHY used for USB3 operations is already enabled.

Change-Id: Idda60c42f1678461e24bf3064dff5dad6dcf2b65
Signed-off-by: Pankaj Dev <pankaj.dev@st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/305764
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
Domain-Review: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
---
 arch/arm64/configs/fragment-02-defconfig-addons.config | 3 +++
 1 file changed, 3 insertions(+)

--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -4,8 +4,11 @@ CONFIG_ARM_SCMI_CPUFREQ=y
 CONFIG_ARM_SMC_WATCHDOG=y
 CONFIG_DYNAMIC_DEBUG=y
 CONFIG_I2C_STM32F7=y
+CONFIG_PHY_STM32_USB2PHY=y
 CONFIG_REGULATOR_ARM_SCMI=y
 CONFIG_RTC_DRV_STM32=y
 CONFIG_SERIAL_STM32=y
 CONFIG_SERIAL_STM32_CONSOLE=y
 CONFIG_STM32_DMA3=y
+CONFIG_USB_DWC3_STM32=y
+CONFIG_USB_STM32_USBH=y
